include/pybind11/detail/native_enum_data.h

```diff
@@ -67,10 +78,11 @@ class native_enum_data {
     std::type_index enum_type_index;
 
 private:
-    object parent_scope;
+    handle parent_scope;
     str enum_name;
     str native_type_name;
     std::string class_doc;
+    capsule enum_info;
```

Comment:
`capsule metabind_info;`?

Reply:
It's not metabind-specific and is helpful for cleanup even when metabind isn't being used